src/components/button-toggle/button-toggle.ts

```diff
@@ -0,0 +1,327 @@
+import {
+  Component,
+  ContentChildren,
+  Directive,
+  EventEmitter,
+  HostBinding,
+  HostListener,
+  Input,
+  OnInit,
+  Optional,
+  Output,
+  QueryList,
+  ViewEncapsulation,
+  forwardRef
+} from '@angular/core';
+import {Observable} from 'rxjs/Observable';
+import {
+  MdUniqueSelectionDispatcher
+} from '@angular2-material/core/coordination/unique-selection-dispatcher';
+
+
+
+var _uniqueIdCounter = 0;
+
+/** A simple change event emitted by either MdButtonToggle or MdButtonToggleGroup. */
+export class MdButtonToggleChange {
+  source: MdButtonToggle;
+  value: any;
+}
+
+/** Exclusive selection button toggle group. */
```

Comment:
I'd expand this comment to include something like "Behaves like a radio-button group."